example/conf.py

```diff
@@ -106,6 +106,7 @@
 
 html_show_sphinx = False
 html_show_source = False
+html_add_permalinks = '#'
```

Comment:
How is this related to `Enhanced titles`? I don't even know what it does, maybe an explanation in the commit message or in a comment could help.

Reply:
miuauqs